pom.xml

```diff
@@ -81,12 +81,15 @@
         <hibernate-search.version>7.2.3.Final</hibernate-search.version>
 
         <!-- Make sure to check compatibility between these 2 gRPC components before upgrade -->
-        <grpc.version>1.69.1</grpc.version> <!-- when updating, verify if com.google.auth should not be updated too -->
+        <grpc.version>1.69.1</grpc.version> <!-- when updating, verify if following versions should not be updated too: -->
+                                            <!-- com.google.auth -->
+                                            <!-- perfmark.version https://central.sonatype.com/artifact/io.grpc/grpc-core  -->
         <grpc-jprotoc.version>1.2.2</grpc-jprotoc.version>
         <protoc.version>3.25.5</protoc.version>
         <protobuf-java.version>${protoc.version}</protobuf-java.version>
         <protobuf-kotlin.version>4.29.3</protobuf-kotlin.version>
         <proto-google-common-protos.version>2.56.0</proto-google-common-protos.version>
+        <perfmark.version>0.27.0</perfmark.version><!-- dependency of io.grpc:grpc-core not managed in io.grpc:grpc-bom -->
```

Comment:
@ppalaga, to me it seems that `perfmark` should be managed in `io.grpc:grpc-bom`. Can you please create io.grpc issue/PR?